app/Http/routes.php

```diff
@@ -32,45 +32,24 @@
 	Route::any('/noSessionTimeout', 'GenericController@noSessionTimeout');
 	Route::any('/api/getNotifications', 'GenericController@getNotifications');
 	Route::any('/api/markNotificationsAsRead', 'GenericController@markNotificationsAsRead');
-	
-	Route::get('/api/getUserProfile', 'UserController@getUserProfile');
-	Route::get('/api/getDashboardData', 'UserController@getDashboardData');
+
+	Route::get('/api/getMemberProfile', 'MemberController@getMemberProfile');
+	Route::get('/api/getDashboardData', 'MemberController@getDashboardData');
 
 	Route::get('/api/getNews', 'NewsController@getNews');
 	Route::get('/api/getNewsById', 'NewsController@getNewsById');
 
 	// Personal routes..
-	Route::post('/api/changeEmail', 'UserController@changeEmail');
-	Route::post('/api/changePassword', 'UserController@changePassword');
-	Route::post('/api/editBio', 'UserController@editBio');
-	Route::post('/api/uploadUserAvatar', 'UserController@uploadUserAvatar');
-
-	// Antennae management..
-	Route::group(['middleware' => 'checkAccess:antennae_management'], function() {
-		Route::get('/api/getAntennae', 'AntennaController@getAntennae');
-		Route::post('/api/saveAntenna', 'AntennaController@saveAntenna');
-		Route::get('/api/getAntenna', 'AntennaController@getAntenna');
-	});
-
-	// Working groups..
-	Route::group(['middleware' => 'checkAccess:working_groups'], function() {
-		Route::get('/api/getWorkingGroups', 'WorkingGroupController@getWorkingGroups');
-		Route::post('/api/saveWorkGroup', 'WorkingGroupController@saveWorkGroup');
-		Route::get('/api/getWorkGroup', 'WorkingGroupController@getWorkGroup');
-	});
-
-	// Departments..
```

Comment:
Why remove?
Please do NOT remove existing features. You don't like them, disable them from modules management.
It will break the app for people who are actually using this.